components/ClaimCard.tsx

```diff
@@ -14,6 +19,7 @@ export const ClaimCard: React.FC = () => {
           'Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.',
         ]}
       />
+      <h1>{uniqueNumber}</h1>
```

Comment:
instead of passing the uniqueNumber to a component, let's use the logger library @lomky just added to log it on the backend whenever we make a request to the API gateway! we don't otherwise need the UN on the front-end so better not to display it.

i'd suggest in terms of just getting this working on test, just strip it out for now, and use the logger library in the next PR

Reply:
Oo yeah, good catch.